Fruitties/iosApp/iosApp/ContentView.swift

```diff
@@ -67,37 +67,37 @@ struct FruittieView: View {
 }
 
 class UIModel: ObservableObject {
```

Comment:
It kind of looks like the new way to do this is to just use the macro `@Observable` 

https://developer.apple.com/documentation/swiftui/migrating-from-the-observable-object-protocol-to-the-observable-macro

Reply:
I can do this -- it requires updating the app to support iOS 17 (minimum)